renovate.json

```
@@ -1,5 +1,3 @@
 {
-  "extends": [
-    "config:base"
-  ]
+    "extends": ["config:base"]
```

Let's change this to the Pliancy default:

```suggestion
    "extends": ["github>pliancy/.github"]
```